.changeset/polite-colts-joke.md

```diff
@@ -0,0 +1,6 @@
+---
+'@commercetools-uikit/select-input': minor
+'@commercetools-uikit/select-utils': minor
+---
+
+provide min and max width props to the select input component
```

Comment:
We should enumerate the new props here and explain they control the `min` and `max` width of the select dropdown menu.